src/Shell/QueueShell.php

```diff
@@ -65,6 +67,22 @@ public function initialize() {
 			}
 		}
 
+		if (Configure::read('Queue.log')) {
```

Comment:
I am still not too happy about this.
FileLog hardcoded here means you cannot use it on any system that cannot use File logging (multi server or heroku etc).
Why not manually doing this in your own config/boostrap - or putting in into the plugins bootstrap and either say bootstrap=>true/false when loading the plugin?